integration-cli/requirements_test.go

```diff
@@ -69,6 +69,10 @@ func UnixCli() bool {
 	return isUnixCli
 }
 
+func GitHubActions() bool {
+	return os.Getenv("GITHUB_ACTIONS") == "true"
```

Comment:
strconv.ParseBool

Reply:
Ah, dang; I recall I commented on this to use "non-empty" (as I don't think we'll ever gonna use `GITHUB_ACTIONS=false`); https://github.com/moby/moby/pull/44034#discussion_r958234696

Looks like it was inadvertently marked as "resolved"; does that suggestion work for you, @AkihiroSuda ?

Reply:
non-empty also sgtm